src/components/Unknown/BuildScreen/Filter/FilterItem/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme } from '@mui/material';
+import { createStyles, makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    wrapper: {
+      backgroundColor: theme.palette.backgroundGrey,
+      borderRadius: theme.shape.borderRadius,
+      margin: theme.spacing(2),
+      cursor: 'pointer',
+    },
+    button: {
+      width: '100%',
+      height: '100%',
+      borderRadius: theme.shape.borderRadius,
+      padding: theme.spacing(2),
+    },
```

Comment:
We don't have button in theme?

Reply:
This is a customized base button, changing it in theme will lead to distortion of all other buttons